node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/CapacityPolicies.java

```diff
@@ -82,6 +82,20 @@ private NodeResources specifiedOrDefaultNodeResources(Optional<NodeResources> re
         }
     }
 
+    private NodeResources defaultNodeResources(ClusterSpec.Type clusterType) {
+        if (zone.system() == SystemName.PublicCd && clusterType == ClusterSpec.Type.admin && zone.environment() != Environment.prod)
+            return new NodeResources(1, 3, 50);
+
+        if (zone.system() == SystemName.cd && zone.environment() == Environment.dev && zone.region().value().equals("cd-us-west-1"))
+            return new NodeResources(1, 4, 50);
```

Comment:
no idea why this particular zone requires 4G RAM, sounds wrong - should not be needed - you can leave be and we can remove once we know why we need this - @hmusum ?

Reply:
No need for this, just remove it

Reply:
Done, thanks

Reply:
Given that we're not capping cpu in cd non-prod now I guess we can also remove the next clause,
        if (zone.system() == SystemName.cd && zone.environment() == Environment.test || zone.environment() == Environment.staging)
            return clusterType == ClusterSpec.Type.admin ? new NodeResources(1, 3, 50)
                                                         : new NodeResources(4, 4, 50);
?

Reply:
We would need more Docker hosts, since we need more memory for the non-admin case. The admin clause can be removed.